track/templates/includes/fr/header.html

```diff
@@ -0,0 +1,7 @@
+<div class="container mx-auto items-center sm:w-4/5 xl:w-3/5 mt-6">
+	<h1 class="text-4xl sm:text-5xl">Vérifier la conformité</h1>
+	<h2 class="mb-4 sm:mb-6 text-xl sm:text-2xl">Dernière mise à jour {{ scan_date | display_date("fr") }}</h2>
+	<p class="text-lg mb-6">Ce tableau indique dans quelles proportions les sites et services en ligne gouvernementaux respectent les bonnes pratiques en matière de sécurité Web décrites dans l’Avis de mise en œuvre de la Politique sur la technologie de l’information (AMPTI) : Utiliser le protocole HTTPS pour des connexions Web sécurisées.</p>
```

Comment:
French ITPIN name is `Mise en œuvre de HTTPS pour les connexions Web sécurisées`